modules/commandhandler/index.js

```diff
@@ -0,0 +1,81 @@
+const config = require('../../config.js');
+const load = require('load-script2');
+
+/**
+ * The jitsi-meet api.
+ */
+let api;
```

Comment:
I moved the api reference to the command handler module because I don't really find a reason it needs to be in app.js file. Will this be alright?